tests/source/issue-5609.rs

```diff
@@ -1,4 +1,4 @@
-// rustfmt-config: issue-5609.toml
+// rustfmt-ignore_missing_submod: true
 
 mod missing_submod;
 
```

Comment:
The `source` file actually isn't necessary either. Since we're not doing any formatting to the file the `target` file is enough for this test.

Reply:
Actually, on second thought, I think keeping the `source` test is good to demonstrate that formatting is still happening. Can we explicitly set `skip_children=false`. `skip_children=false` is the current default, and when this is set we'll try to resolve sub modules. Again, this is the current default, but formatting would work just as well if `skip_children=true`, since we then wouldn't try to recursively resolve children, and therefore the `ignore_missing_submod` option wouldn't have any affect.

In case we change the default behavior of `skip_children` to be `true` in the future explicitly setting `false` will ensure this test keeps working as intended.